packages/web/src/dev/ValidatorsListRow.tsx

```diff
@@ -12,6 +12,8 @@ import { cutAddress, formatNumber } from 'src/utils/utils'
 const unknownGroupName = 'Unnamed Group'
 const unknownValidatorName = 'Unnamed Validator'
 
+const localStoragePinnedKey = 'pinnedValidators'
```

Comment:
this is the same key as in the other file right? 

might be good to define once and import